week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,9 +1,34 @@
-function setAlarm() {}
+function setAlarm() {
```

Comment:
This has the interesting effect that when the alarm is going off, the pages shows `00:01` - I would have expected it to show `00:00` - can you give that a go? :)